docs/test-content.md

```diff
@@ -16,12 +16,8 @@ Because c2d is a fairly expensive operation, the service app gathers `serviceAck
       "serviceRunId": "ca3ceadb-ac57-4cca-8f9b-c5d1e4dc189d",
       "pairingId": "3887e97e-6f06-484b-a5bf-1753913866d3",
       "serviceAcks": [
```

Comment:
`serviceAckIds`?

Reply:
I'm going to keep it as serviceAcks for now.  There's a chance that I'll have to expand it to include more data per ack (timestamps specifically).